src/pages/docs/administration/managing-infrastructure/server-configuration-and-file-storage/index.md

```diff
@@ -49,6 +49,10 @@ The Octopus Server stores files in the following folders by default:
     - This folder was added in **Octopus 2021.1**
     - This is where imported zip files are stored when using the [Export/Import Projects feature](/docs/projects/export-import).
     - See this [page](/docs/administration/managing-infrastructure/server-configuration-and-file-storage/moving-octopus-server-folders/#MovingOctopusServerfolders-Imports) on how to move the Octopus Server imports folder
+- `C:\Octopus\EventExports`
+    - This folder was added in **Octopus 2023.3**
+    - This is where audit log archive zip files are stored when using the [Archived audit logs feature](/docs/security/users-and-teams/auditing/index.md#archived-audit-events).
```

Comment:
I think with Astro (the new site "engine"), you dont need to include the index.md extension for links
```suggestion
    - This is where audit log archive zip files are stored when using the [Archived audit logs feature](/docs/security/users-and-teams/auditing/#archived-audit-events).
```